x/sequencer/keeper/hook_listener.go

```diff
@@ -58,12 +70,13 @@ func (hook rollappHook) FraudSubmitted(ctx sdk.Context, rollappID string, height
 	// unbond all other bonded sequencers
 	sequencers := hook.k.GetSequencersByRollappByStatus(ctx, rollappID, types.Bonded)
 	for _, sequencer := range sequencers {
-		err := hook.k.forceUnbondSequencer(ctx, sequencer.SequencerAddress)
-		if err != nil {
-			return err
-		}
+		hook.k.startUnbondingPeriodForSequencer(ctx, &sequencer)
```

Comment:
on fraud, i've changed the bonded seqs to start `unbonding` instead of fully unbonded

Reply:
why is that? I think it actually doesn't make sense cause most likely one of those sequencers will be the one starting the forked rollapp (we can look at it as a non-graceful rotation) and in essence we lock them for 21 days.

Reply:
I've changed it as:
* Currently, seqeuncer is coupled with rollapp. even after unbonded.
to sequence a new rollapp, one needs a different sequencer address
* I thought there might be different scenarios where u might need to slash the **non-proposer** (e.g the next proposer)
thought the safe side will be to enforce unbonding period

Reply:
reverted